public/app/features/dashboard/state/initDashboard.ts

```diff
@@ -206,6 +208,25 @@ export function initDashboard(args: InitDashboardArgs): ThunkResult<void> {
     }
 
     // yay we are done
+    if (dashDTO.dashboard.uid) {
+      dashboard.panels.forEach((panel) => {
+        panel.targets.forEach((target) => {
+          let dsSpecific = {};
+          if (target.datasource?.type === 'grafana-azure-monitor-datasource') {
+            dsSpecific = getAzureMonitorEvent(target as AzureMonitorQuery);
+          }
+          reportInteraction('grafana_dashboard_loaded', {
```

Comment:
I agree. Think the event `reportMetaAnalytics` is for the enterprise feature with the same name. But we can just create a new event and add a corresponding handler in the echoservice.

Reply:
I suppose I saw the dashboard load as a user interaction, but I understand how it can be viewed differently than a click

The `reportMetaAnalytics` would not fit our purpose as it's very specific to dashboard views and data requests and we need custom properties to log
```
MetaAnalyticsEventPayload = DashboardViewEventPayload | DataRequestEventPayload
```
Looking at supported types here: https://github.com/grafana/grafana/blob/main/packages/grafana-runtime/src/services/EchoSrv.ts#L73-L86 and the subset that Rudderstack supports: https://github.com/grafana/grafana/blob/e96f67ae2e01cd61a9a3d26442a75d8ba5d674ad/public/app/core/services/echo/backends/analytics/RudderstackBackend.ts#L24

It looks like interaction is the best fit? 
I would not mind creating a new handler, but it seems overkill to me to create a new event type for an MVP as we'd have to touch the Rudderstack backend.

Reply:
yeah, this is a bit of a nit, so we can go ahead with `reportInteraction`